ofl/amiriqurancolored/DESCRIPTION.en_us.html

```diff
@@ -1,6 +1,54 @@
 <p>
-Amiri is a classical Arabic typeface in Naskh style for typesetting books and other running text. Its design is a revival of the beautiful typeface pioneered in early 20th century by Bulaq Press in Cairo, also known as Amiria Press, after which the font is named.
+    Amiri (أميري) is a classical Arabic typeface in Naskh style for typesetting books and
+    other running text.
 </p>
+<p>  
+    Amiri is a revival of the beautiful typeface pioneered in early 20th century by
+    Bulaq Press in Cairo, also known as Amiria Press, after which the font is
+    named.
+</p> 
+<p>    
+    The uniqueness of this typeface comes from its superb balance between the
+    beauty of Naskh calligraphy on one hand, the constraints and requirements of
+    elegant typography on the other. Also, it is one of the few metal typefaces
+    that were used in typesetting the Koran, making it a good source for a digital
+    typeface to be used in typesetting Koranic verses.
+</p>
+<p>    
+    Amiri project aims at the revival of the aesthetics and traditions of Arabic
+    typesetting, and adapting it to the era of digital typesetting, in a publicly
+    available form.
+</p>
+
+<p dir="rtl" lang="ar">
+الخط الأميري خط نسخي موجه لطباعة الكتب و النصوص الطويلة.
+</p>
+
+<p dir="rtl" lang="ar">
+الخط الأميري هو إحياء و محاكاة للخط الطباعي الجميل الذي تميزت به مطبعة بولاق منذ
+أوائل القرن العشرين، و التي عرفت أيضا بالمطبعة الأميرية، و من هنا أخذ الخط اسمه.
+</p>
+
+<p dir="rtl" lang="ar">
+يتميز خط المطابع الأميرية بجماليته و مراعاته لفن الخط العربي، بأسلوب نسخي جميل،
+و في ذات الوقت يراعى متطلبات الطباعة و القيود التي تفرضها، من غير إفراط في جانب
+على حساب الآخر. و لهذا يتميز بمناسبته للصف الطباعي عموما، و لصف الكتب خصوصا. و قد
+استُخدِم هذا الخط في طباعة المصحف الأميري، و هو من المصاحف القليلة التي طبعت بالصف
+المعدني و لم يخطها خطاط بيده، و هذا يجعله مادة خصبة لبناء خط حاسوبي مناسب لصف
+النصوص القرآنية.
+</p>
+
+<p dir="rtl" lang="ar">
+يهدف مشروع الخط الأميري إلى إحياء تقاليد و جماليات الطباعة العربية و موائمتها
+لتقنيات عصر الحواسيب، مع إتاحتها للعموم.
+</p>
+
+<p dir="rtl" lang="ar">
+يمكن الحصول على آخر إصدارات الخط الأميري من موقعه:
+</p>
+
+<p><a href="http://amirifont.org">amirifont.org</a></p>
+
 <p>
 To contribute, see <a href="https://github.com/alif-type/reem-kufi">github.com/alif-type/amiri</a>
```

Comment:
The link is wrong here.

Reply:
https://www.amirifont.org is the preferred option?

Reply:
Ah no you were talking about the github repo, on it.